spi/java/org/openyolo/spi/BaseCredentialQueryReceiver.java

```diff
@@ -55,19 +55,19 @@ protected void processQuery(@NonNull Context context, @NonNull BroadcastQuery qu
         }
 
         // TODO: validate claimed authentication domains
```

Comment:
while we're here, is this a stale TODO that doesn't apply anymore, since we don't have claimed auth domains?